package.json

```diff
@@ -2,9 +2,10 @@
   "name": "jslearn-complited-tasks",
   "version": "1.0.0",
   "description": "",
-  "main": "./src/modules/stack.js",
+  "main": "./src/lib/stack/stack.js",
   "scripts": {
-    "test": "mocha ./src/modules/stack.test.js"
```

Comment:
you don't need to specify each test, you may use this : "test": "mocha \"./{,!(node_modules)/**/}*.test.js\"" instead. It will run all test from all files ended with .test.js , exclude external tests from imported modules from node_modules